greenplum-postgresql.tf

```
@@ -5,45 +5,74 @@
 
 # Specify the following settings
 locals {
+  # Settings for Managed Service for Greenplum® cluster:
+  gp_version  = "" # Set the Greenplum® version. For available versions, see the documentation main page: https://cloud.yandex.com/en/docs/managed-greenplum/.
   gp_password = "" # Set a password for the Greenplum® admin user
+
+  # Settings for Managed Service for PostgreSQL cluster:
+  pg_version  = "" # Set the PostgreSQL version. For available versions, see the documentation main page: https://cloud.yandex.com/en/docs/managed-postgresql/.
   pg_password = "" # Set a password for the PostgreSQL admin user
 
-  # Specify these settings ONLY AFTER the clusters are created. Then run "terraform apply" command again
+  # Specify these settings ONLY AFTER the clusters are created. Then run "terraform apply" command again.
   # You should set up the source endpoint using the GUI to obtain its ID
   gp_source_endpoint_id = "" # Set the source endpoint ID
-  transfer_enabled      = 0  # Value '0' disables creating of transfer before the source endpoint is created manually. After that, set to '1' to enable transfer
+  transfer_enabled      = 0  # Value '0' disables creating of transfer before the source endpoint is created manually. After that, set to '1' to enable transfer.
```

Надо либо:

```suggestion
  transfer_enabled      = 0  # Value '0' disables creating a transfer before the source endpoint is created manually. After that, set to '1' to enable the transfer.
```

Либо:

```suggestion
  transfer_enabled      = 0  # Value '0' disables the transfer creation before the source endpoint is created manually. After that, set to '1' to enable the transfer.
```

В текущем варианте 100% неправильно, нельзя писать отгл. сущ. с ing + of. Я бы второй вариант выбрала, мне кажется, он больше подходит.

---

Исправила.